swarm/src/behaviour.rs

```diff
@@ -130,7 +130,7 @@ pub trait NetworkBehaviour: Send + 'static {
     }
 
     /// A listener closed.
-    fn inject_listener_closed(&mut self, _id: ListenerId) {
+    fn inject_listener_closed(&mut self, _id: ListenerId, _err: Option<std::io::Error>) {
```

Comment:
Could you leave the `Result` for consistency with the rest of the library?

```suggestion
    fn inject_listener_closed(&mut self, _id: ListenerId, reason: Result<(), io::Error>) {
```

Reply:
Sure thing, can do.